### DISCUSSION TITLE
windows install issue

### DISCUSSION THREAD

**Discussion by trw828** [publicsamples/Quetzalcoatl#20] [General]
I have the VST in my DAW but when opening it prompts me to install samples, then when I click that, it asks me to choose the sample archive location, I open file explorer and choose where i want the samples but it won't let me press "open" to choose the sample archive. Does anyone know how to fix this?

**Comment by RickTaylor79** [comment]
I can't reproduce this, can you post a screen shot?

**Reply by trw828** [reply]
Hi, thanks for replying
![image](https://github.com/user-attachments/assets/05300d6d-5953-47fb-afce-01b6791fcd4e)
i press install samples
![image](https://github.com/user-attachments/assets/2247d1a0-56bb-425d-a793-f7a31cf6182a)
i select where i want to install the samples in file explorer and it brings me to this page
![image](https://github.com/user-attachments/assets/d80bf7cf-8774-4828-828a-3f319a22dbdc)
i press to change sample archive location and select where i want to install again but in file explorer the open button doesnt work
![image](https://github.com/user-attachments/assets/0d80b638-fc44-4765-be18-9eb52d608bc6)

**Reply by RickTaylor79** [reply]
Okay,  you need to select the Quetzalcoatl_0_6_6_Samples.hr1 file at this stage.

**Reply by anontheperson** [reply]
> Okay, you need to select the Quetzalcoatl_0_6_6_Samples.hr1 file at this stage.

where do i download that

**Reply by RickTaylor79** [reply]
Here if you're using v0.7.0: https://github.com/publicsamples/Quetzalcoatl/releases/download/0.7.0/Quetzalcoatl_0_7_0_Samples.hr1

Or for 0.6.6:
https://github.com/publicsamples/Quetzalcoatl/releases/download/0.6.6/Quetzalcoatl_0_6_6_Samples.hr1
